<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>V3DCfgWidForAttributeAsContourLines</class>
 <widget class="QWidget" name="V3DCfgWidForAttributeAsContourLines">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Value interval: min:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinContourMin">
       <property name="minimum">
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>max:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinContourMax">
       <property name="minimum">
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Number of contours:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinNumberOfLines">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Contour lines width (pixels):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinContourLinesWidth">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Contour lines color:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Set font color</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnColor</sender>
   <signal>clicked()</signal>
   <receiver>V3DCfgWidForAttributeAsContourLines</receiver>
   <slot>onColorChoose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinContourLinesWidth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>V3DCfgWidForAttributeAsContourLines</receiver>
   <slot>onUserMadeChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinNumberOfLines</sender>
   <signal>valueChanged(int)</signal>
   <receiver>V3DCfgWidForAttributeAsContourLines</receiver>
   <slot>onUserMadeChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinContourMax</sender>
   <signal>valueChanged(double)</signal>
   <receiver>V3DCfgWidForAttributeAsContourLines</receiver>
   <slot>onUserMadeChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinContourMin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>V3DCfgWidForAttributeAsContourLines</receiver>
   <slot>onUserMadeChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onUserMadeChanges()</slot>
  <slot>onColorChoose()</slot>
 </slots>
</ui>
